commcare_connect/opportunity/migrations/0074_opportunityaccess_completed_learn_date.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 4.2.5 on 2025-05-08 10:07
+import logging
+
+from django.db import migrations, models
+from django.db.models import Count, Min
+
+from commcare_connect.utils.itertools import batched
+
+logger = logging.getLogger(__name__)
+
+
+def back_fill_completed_learn_date(apps, schema_editor):
+    Opportunity = apps.get_model("opportunity", "Opportunity")
+    OpportunityAccess = apps.get_model("opportunity", "OpportunityAccess")
+    CompletedModule = apps.get_model("opportunity", "CompletedModule")
+
+    BATCH_SIZE = 50
```

Comment:
This batch seems unnecessarily small but fine for now. Next time I would think 100 or even 1000.